test/unit/features/directives/on.spec.js

```diff
@@ -460,6 +460,20 @@ describe('Directive v-on', () => {
     expect(spy).toHaveBeenCalled()
   })
 
+  it('should throw a warning if native modifier is used on native HTML element', () => {
+    vm = new Vue({
+      el,
+      template: `
+        <button @click.native="foo"></button>
+      `,
+      methods: { foo: spy },
+    })
+
+    triggerEvent(vm.$el, 'click')
+    expect(`The .native modifier for v-on is only valid on components but it was used on <${tag}>.`).toHaveBeenWarned()
```

Comment:
Should the test assertion have `${tag}` replaced with `button`?

```suggestion
    expect(`The .native modifier for v-on is only valid on components but it was used on <button>.`).toHaveBeenWarned()
```

Reply:
Thanks, you're totally right 👍